### PR TITLE
reports: Add ZAP version to HTML and PDF reports

### PR DESCRIPTION
**Pull request by dailz-c**
Added owasp zap version to html and pdf reports

### CODE REVIEW

**Comment by psiinon** [comment]
Its failiong a format check - you probably need to do a `spotlessApply`.
Its probably going to fail the unit tests as well 😁

File: addOns/reports/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Added
+- Add Zap version to html and pdf exports
```
Comment (thc202) [inline addOns/reports/CHANGELOG.md:8]: `- Add ZAP version to HTML and PDF reports.`

**Comment by psiinon** [comment]
Run `./gradlew :addOns:reports:spotlessApply` to fix these violations.

File: addOns/reports/CHANGELOG.md (hunk 1 above)
Comment (dailz-c) [reply addOns/reports/CHANGELOG.md]: Updated the message

**Comment by dailz-c** [comment]
Ran spotlessApply and update the change log!

File: addOns/reports/src/main/zapHomeFiles/reports/high-level-report/report.html [hunk 2]
```diff
@@ -42,8 +42,9 @@ <h1>
 		<div class="row">
 			<em
 				th:text="#{report.generated(${#dates.format(new java.util.Date(), 'EEE, d MMM yyyy HH:mm:ss')})}"
-				th:class="'text-right'"></em>
-
+				th:class="'text-right'">Date, time</em> <em
+				th:text="#{report.owaspZapVersion(${zapVersion})}"
+				th:class="'text-right'">OWASP Version</em>
```
Comment (kingthorin) [inline addOns/reports/src/main/zapHomeFiles/reports/high-level-report/report.html:47]: This should include ZAP.

@psiinon @thc202 should we just use `ZAP Version` for reports? (Ex: without "OWASP").

Comment (thc202) [reply addOns/reports/src/main/zapHomeFiles/reports/high-level-report/report.html:47]: Sounds good.

Comment (kingthorin) [reply addOns/reports/src/main/zapHomeFiles/reports/high-level-report/report.html:47]: Okay let’s go with just `ZAP Version` throughout.

**Comment by kingthorin** [comment]
It seems there are other tests that need fixing by accommodating the new output.

**Comment by dailz-c** [comment]
Updated the tags to have a standard name and fixed the updated the two tests that were failing when I ran it locally

File: addOns/reports/src/main/zapHomeFiles/reports/risk-confidence-html/report.html [hunk 3]
```diff
@@ -26,6 +26,8 @@ <h1 data-th-text="${reportTitle}">Report Title</h1>
 				data-th-text="${#dates.format(new java.util.Date(), #messages.msg('report.template.generatedDatetime'))}">on
 				date, at time</span>
 		</p>
+		<p data-th-text="#{report.owaspZapVersion(${zapVersion})}">ZAP Version</th:block>
```
Comment (thc202) [inline addOns/reports/src/main/zapHomeFiles/reports/risk-confidence-html/report.html:29]:
```suggestion
		<p data-th-text="#{report.owaspZapVersion(${zapVersion})}">ZAP Version
```

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you! How would you like to be credited (e.g. name, handle)?
https://www.zaproxy.org/docs/desktop/credits/#zap-extended-team

**Comment by dailz-c** [comment]
@thc202 whatever is standard works for me!

**Comment by kingthorin** [comment]
It's kind of a mixed bag, that's why we provide the link and ask people :wink:

**Comment by dailz-c** [comment]
@kingthorin Chris Dailey (@dailz-c) works then!

**Pull request merged**